tests/all_tests.rb

```diff
@@ -1,8 +1,9 @@
-Dir.chdir(File.dirname(__FILE__)) do
-  Dir.glob('**/test_*.rb') do |test_case|
-    require "#{File.expand_path(File.dirname(__FILE__))}/#{test_case}"
-  end
+Dir.glob('tests/**/test_*.rb') do |test_case|
+  require File.expand_path(test_case)
```

Comment:
How about this to avoid 39 calls to `File.expand_path`:

```ruby
require test_case[6..-1]
```

Reply:
I was a bit unsure whether the intent of the array slicing would be clear to the reader; instead I've managed to get `Dir.glob` spitting out full absolute paths that can be directly `require`d without further manipulation. I've pushed an amended commit, see what you think?

Reply:
In fact `all_tests.rb` felt pointlessly small now so I've gotten rid of it and pushed the glob up to the rake task.